docs/standard/clr.md

```diff
@@ -54,23 +54,6 @@ Language compilers and tools expose the runtime's functionality in ways that are
 
 - Use of delegates instead of function pointers for increased type safety and security. For more information about delegates, see [Common Type System](../../docs/standard/base-types/common-type-system.md).
 
```

Comment:
```suggestion
## CLR versions	

The .NET Framework or .NET Core version number doesn't necessarily correspond to the version number of the CLR it includes. For a list of .NET Framework versions and their corresponding CLR versions, see [.NET Framework versions and dependencies](../../framework/migration-guide/versions-and-dependencies.md). For a list of .NET Core versions, see [Download .NET Core](https://dotnet.microsoft.com/download/dotnet-core).
```
The only problem is that the linked page for .NET Core doesn't show which CLR version is included, which is what this section seems to be about.

Reply:
.NET Core does not have a concept of "included CLR version". It is a uniform product version all up.

Reply:
Okay, got it. I updated my suggestion to reflect that. What do you think?

Reply:
Yes, this looks good to me. Thanks for the suggestion.